MagneticField/GeomBuilder/src/MagGeoBuilderFromDDD.h

```diff
@@ -123,6 +98,6 @@ class MagGeoBuilderFromDDD {
   std::map<int, double> theScalingFactors;
   const magneticfield::TableFileMap* theGridFiles;  // Non-owned pointer assumed to be valid until build() is called
 
-  static bool debug;
+  const bool debug;
```

Comment:
My understanding is that Carl has decided to move away from a single static used everywhere to a model where the flag is percolated. I am ok with that as far as design is concerned.

Reply:
For the debugging strategy, moving to using `#ifdef EDM_ML_DEBUG` instead of a debug flag could even be considered. It would go along with `LogTrace` and would remove any performance penalty of storing debug flags and of processing `if (debug)` statements but would not allow turning on debugging in a program running in production.

Nicola has a more complete view of the debugging needs of this code, so I would like to leave the debugging design to him. I think I should stop fiddling with it.